src/boot_O2_g3/fault.c

```diff
@@ -762,21 +763,23 @@ void Fault_SetOptionsFromController3(void) {
     input3 = &sFaultContext->padInput[3];
 
     if (CHECK_BTN_ALL(input3->press.button, 0x80)) {
-        faultCustomOptions = faultCustomOptions == 0;
+        faultCustomOptions = !faultCustomOptions;
     }
 
     if (faultCustomOptions) {
         graphPC = sGraphThread.context.pc;
         graphRA = sGraphThread.context.ra;
         graphSP = sGraphThread.context.sp;
-        if (CHECK_BTN_ALL(input3->press.button, BTN_R)) {
+        if (CHECK_BTN_ALL(input3->cur.button, BTN_R)) {
+            static u32 faultCopyToLog;
+
             faultCopyToLog = !faultCopyToLog;
```

Comment:
imo it looks cleaner this way
```suggestion
            static u32 faultCopyToLog = !faultCopyToLog;
```

Reply:
Apparently this is not allowed.
![image](https://user-images.githubusercontent.com/31974442/141330473-0d7610d9-f3be-4b39-bd54-5a28ea174367.png)

Reply:
Oh right, I misread what was going on here, sorry. Now that I read it properly I think it's best left as-is.